.github/ISSUE_TEMPLATE/4-release_checklist.md

```diff
@@ -10,47 +10,47 @@ assignees: ''
   - [ ] All tests pass in the ["GMT Legacy Tests" workflow](https://github.com/GenericMappingTools/pygmt/actions/workflows/ci_tests_legacy.yaml)
   - [ ] All tests pass in the ["GMT Dev Tests" workflow](https://github.com/GenericMappingTools/pygmt/actions/workflows/ci_tests_dev.yaml)
   - [ ] All tests pass in the ["Doctests" workflow](https://github.com/GenericMappingTools/pygmt/actions/workflows/ci_doctests.yaml)
-  - [ ] Deprecations and related tests are removed for this version by running `grep --include="*.py" -r 'remove_version="vX.Y.Z"' pygmt` from the base of the repository
-- [ ] Update warnings in `pygmt.show_versions()` as well as notes in [Common installation issues](https://www.pygmt.org/dev/install.html#not-working-transparency)
-      and [Testing your install]((https://www.pygmt.org/dev/install.html#testing-your-install) regarding GMT-Ghostscript incompatibility
+- [ ] Update warnings in `pygmt/_show_versions.py` as well as notes in
+      [Not working transparency](https://www.pygmt.org/dev/install.html#not-working-transparency)
+      regarding GMT-Ghostscript incompatibility
 - [ ] Reserve a DOI on [Zenodo](https://zenodo.org) by clicking on "New Version"
-- [ ] Review the ["PyGMT Team" page](https://www.pygmt.org/dev/team.html)
-- [ ] Finish up 'Changelog entry for v0.x.x' Pull Request:
-  - [ ] Add a new entry in `doc/_static/version_switch.js` for documentation switcher
-  - [ ] Update `CITATION.cff` and BibTeX at https://github.com/GenericMappingTools/pygmt#citing-pygmt
-    - [ ] Update authorship list
-    - [ ] Update DOI (and url for BibTeX)
-    - [ ] Update version
-    - [ ] Update date released
-  - [ ] Add the documentation link `doc/minversions.md`
-  - [ ] Add minimum required version information `doc/minversions.md`
-  - [ ] Copy draft changelog from Release Drafter and edit it to look nice ([see maintainers guide for details](https://www.pygmt.org/dev/maintenance.html#updating-the-changelog))
+- [ ] Finish up the "Changelog entry for v0.x.x" Pull Request (Use the previous changelog PR as a reference)
+- [ ] Run `make codespell` to check common misspellings. If there are any, either fix them or add them to `ignore-words-list` in `pyproject.toml`
+- [ ] Draft the announcement on https://hackmd.io/@pygmt
 
 **Release**:
+
 - [ ] At the [PyGMT release page on GitHub](https://github.com/GenericMappingTools/pygmt/releases):
   - [ ] Edit the draft release notes with the finalized changelog
   - [ ] Set the tag version and release title to vX.Y.Z
+  - [ ] Make sure "Set as the latests release" and "Create a discussion for this release" are selected
   - [ ] Make a release by clicking the 'Publish Release' button, this will automatically create a tag too
 - [ ] Manually upload the pygmt-vX.Y.Z.zip and baseline-images.zip files to https://zenodo.org/deposit, ensure that it is filed under the correct reserved DOI
```

Comment:
@weiji14 One question for you: It seems the zip file downloaded from https://github.com/GenericMappingTools/pygmt/releases/tag/v0.13.0 has a name like `pygmt-0.13.0.zip`, but the zip file on Zenodo has a name like `pygmt-v0.13.0.zip` (the extra `v`). Did you manually rename `pygmt-0.13.0.zip` to `pygmt-v0.13.0.zip` when you uploaded previous versions?

Reply:
Yes, good catch. I always needed to rename it to add an extra 'v'.

Reply:
Any specific reason for the renaming? The zip file is called `pygmt-v0.13.0.zip`, but the directory in the zip file is `pygmt-0.13.0.zip`.

Reply:
I see that this was included in the original release checklist at https://github.com/GenericMappingTools/pygmt/pull/446/commits/f2842c03d94fca0826fc42e23426da5cfb0f752c. Can't remember now since it was 4 years ago, but it might be that our tags follow `v0.X.Y`, so I decided to use `pygmt-v0.X.Y.zip` for the filename, and just kept it to be consistent ever since.

Reply:
Just checked [scipy](https://zenodo.org/records/13352243), [xarray](https://zenodo.org/records/11183201) and [pandas](https://zenodo.org/records/10957263) on Zenodo. It seems they all zenodo register the DOI **after** making a release. Taking pandas as an example, the zip file name is `pandas-dev/pandas-v2.2.2.zip`, and the directory inside is `pydata-xarray-da9ff0a`. So I guess there is no guide about the best names for the zip file and the directory.

I'll mention the renaming in the checklist.